webview-ui/src/components/IncidentTable/IncidentTable.tsx

```diff
@@ -44,7 +44,13 @@ export const IncidentTable: FC<IncidentTableProps> = ({
     <>
       <Card isPlain>
         <CardHeader
-          actions={{ actions: <GetSolutionDropdown incidents={incidents} scope="in-between" /> }}
+          actions={
```

Comment:
Cleaner option could be:

actions={!isReadOnly && {
  actions: <GetSolutionDropdown incidents={incidents} scope="in-between" />
}}

Reply:
tried this. It specifically wanted `undefined` or the `{ actions: ... }` so I updated it at the bottom of the table.